### PR TITLE
Exposed getter for protocolVersion in HttpHeaders.

### PR DESCRIPTION
**Pull request by bkonyi**

### CODE REVIEW

File: lib/src/http_headers_impl.dart [hunk 1]
```diff
@@ -14,7 +14,7 @@ import 'http_exception.dart';
 
 class HttpHeadersImpl implements HttpHeaders {
   final Map<String, List<String>> headers;
-  final String protocolVersion;
+  final String _protocolVersion;
```
Comment (kevmoo) [inline lib/src/http_headers_impl.dart]: There's no problem with implementing a read-only property w/ a final field...right?

**Review by kevmoo — COMMENTED** [review]
drive-by comment

Also: any test coverage for this?

File: lib/src/http_headers_impl.dart (hunk 1 above)
Comment (bkonyi) [reply lib/src/http_headers_impl.dart]: That's a good point. I'll remove the explicit getter and just expose the field in HttpHeaders.

**Comment by bkonyi** [comment]
There is test coverage, just not in this PR. One of the outstanding PRs is importing `http_headers_impl.dart` to check this field, so that will be updated once this is landed.

**Comment by kevmoo** [comment]
ack!

**Review by zanderso — APPROVED** [review]

**Pull request merged**